### PR TITLE
Iterate over all people and fetch publications

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes sul-dlss/rialto#272

### CODE REVIEW

**Comment by justinlittman** [comment]
Need to be able to resume if ever occurs?

**Comment by justinlittman** [comment]
Need to either create data directory or document that it should already exist.

File: lib/rialto/etl/cli/publications.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          end
+        end
+
+        private
+
+        def load_row(row, count)
+          sunetid = row[:sunetid]
+          # Some people don't have sunetids
+          sunetid = "row-#{count}" if sunetid.empty?
+          puts "Retrieving publications for: #{sunetid}"
```
Comment (justinlittman) [inline lib/rialto/etl/cli/publications.rb]: Why don't we skip instead of searching?

Comment (jcoyne) [reply lib/rialto/etl/cli/publications.rb]: Because we're searching by first & last name, not by sunetid.

**Comment by jcoyne** [comment]
@justinlittman can you re-review?

File: lib/rialto/etl/cli/publications.rb (hunk 1 above)
Comment (justinlittman) [reply lib/rialto/etl/cli/publications.rb]: Then we should use profile id, which is our unique identifier for a person. Can either add as a separate field in CSV or extract from existing URI field.

**Comment by justinlittman** [comment]
@jcoyne I'm in the process of testing. However, required getting more of a dev environment set up. Working on it.

File: lib/rialto/etl/cli/publications.rb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
```
Comment (justinlittman) [inline lib/rialto/etl/cli/publications.rb]: Also, it would be really helpful to print the count here so that it is easier to monitor progress.

**Comment by justinlittman** [comment]
Other than above, completed code review and testing (ran though 500 researchers).

**Comment by mjgiarlo** [comment]
@jcoyne Looks like the recent merges caused a conflict in this branch.

**Comment by mjgiarlo** [comment]
@justinlittman Looks good. As soon as the build is :green_heart:, hit me back and I will be happy to merge.

**Pull request merged**